git-hg/mercurialToGit.sh

```diff
@@ -54,6 +54,32 @@ HG_REPO=$1
 GITHUB_REPO=$(echo "$HG_REPO" | cut -d/ -f2)
 BRANCH=${2:-master}
 
+# Determine latest build TAG_SEARCH, finding tags beginning with "jdk-" followed by ... "+" ...
+if [ "$HG_REPO" == "jdk/jdk" ]
+then
+  TAG_SEARCH="jdk-*+*"
+elif [ "$HG_REPO" == "jdk-updates/jdk12u" ]
+then
+  TAG_SEARCH="jdk-12*+*"
+elif [ "$HG_REPO" == "jdk-updates/jdk11u" ]
+then
+  TAG_SEARCH="jdk-11*+*"
+else
+  echo "Unable to determine latest build tag TAG_SEARCH for repo: $HG_REPO"
+  exit 1
+fi
+
+# JDK11+ tag sorting:
+# We use sort and tail to choose the latest tag in case more than one refers the same commit.
+# Versions tags are formatted: jdk-V[.W[.X]]+B; with V, W, X, B being numeric.
+# Transform "-" to "." in tag so we can sort as: "jdk.V[.W[.X]]+B"
+# First, sort on build number (B):
+jdk11plus_tag_sort1="sort -t+ -k2n"
+# Second, (stable) sort on (V), (W), (X):
+jdk11plus_tag_sort2="sort -t. -k2n -k3n -k4n -s"
+jdk11plus_sort_tags_cmd="grep -v _merge | sed 's/jdk-/jdk./g' | $jdk11plus_tag_sort1 | $jdk11plus_tag_sort2 | sed 's/jdk./jdk-/g'"
```

Comment:
Got it - would be nice to show an example of some tags, so new developers can see what you're achieving here

Reply:
sure will add

Reply:
added example